R/exposure_metrics.R

```diff
@@ -111,6 +111,9 @@ calc_auc_from_regimen <- function(
   if(target_design$type == "auc24") {
     target_time <- c(target_time - 24, target_time)
   }
+  if(target_design$type == "auc12") {
```

Comment:
```suggestion
  } else if (target_design$type == "auc12") {
```

Reply:
some of these conditions could be refactored as switches, but for now i think it's fine. we can do that later if it gets too unwieldy.